account/src/main/java/com/caldevsupplychain/account/service/AccountServiceImpl.java

```diff
@@ -1,19 +1,5 @@
 package com.caldevsupplychain.account.service;
 
-import java.util.List;
```

Comment:
You intellij is reordering imports in a diff order. Can you change the setting to keep it consistent?
<img width="1207" alt="screen shot 2017-11-12 at 10 28 20 am" src="https://user-images.githubusercontent.com/31681641/32701991-7b0b8cac-c794-11e7-8742-f53caf1ec704.png">
<img width="927" alt="screen shot 2017-11-12 at 10 28 46 am" src="https://user-images.githubusercontent.com/31681641/32701993-894ea718-c794-11e7-8d08-a5ccbfa449a9.png">
<img width="935" alt="screen shot 2017-11-12 at 10 28 57 am" src="https://user-images.githubusercontent.com/31681641/32701997-923f4544-c794-11e7-88c5-be865faeeaa4.png">